doxia-core/pom.xml

```diff
@@ -59,6 +59,11 @@ under the License.
       <artifactId>commons-lang3</artifactId>
       <version>3.8.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-text</artifactId>
+      <version>1.3</version>
```

Comment:
Why such an old version of commons-text?

Reply:
Because everything newer requires Java 8. :-(

Reply:
That's fine since some time ago, I believe. You may require Java 8 in plugins and components etc for the Maven eco system